dashboard/app/models/pd/international_opt_in.rb

```diff
@@ -39,8 +45,6 @@ def self.required_fields
       :first_name,
       :last_name,
       :gender,
-      :school_name,
-      :school_city,
```

Comment:
Why are we removing these from the required fields? If the generic-case countries still need them, we should keep them required for those countries.

Reply:
Good catch. Originally removed these because they would break any country that wasn't using one of them. [But I've just added an `else` block to the dynamic required fields](https://github.com/code-dot-org/code-dot-org/pull/42809/commits/42c72719dfd32ea5d24317a5f66a6275328c8b25#diff-d60db3d9f9948805fed70027906e14a975c8ddb1c0c683d8d3de8fd09f64c260R141) so in the base case these two fields will be required 👍

Reply:
Thanks! It looks like every country requires a school city though, so can we keep it in the required fields?

Reply:
ah sorry, I meant school name, not city!